automatic/PDFCreator/update.ps1

```diff
@@ -0,0 +1,24 @@
+import-module au
+
+$url = 'http://download.pdfforge.org/download/pdfcreator/PDFCreator-stable?download'
+
+function global:au_SearchReplace {
+    @{
+        'tools\ChocolateyInstall.ps1' = @{
+            "(^[$]checksum\s*=\s*)('.*')"   = "`$1'$($Latest.Checksum32)'"
+        }
+     }
+}
+
+function global:au_GetLatest {
+    Write-Host $url
+
+    $temp_file = $env:TEMP + '\PDFCreator.exe'
+    Invoke-WebRequest $url -OutFile $temp_file
+
+    $version = (Get-Command $temp_file).Version
```

Comment:
Its a good find. I had a same problem with LibreOffice - on site it was one version and it adds revision number. It was important to recognize what you had in order not to download if not needed. However, you solution downloads too, so it doesn't make a difference. But its great soluion when its very hard to parse the version or if its impossible.

Just go with the site 2.4 or blog 2.4.0 (site is probably what is expected by users tbh). Its better not to add revision as it can be used for choco fix.